libs/viewer/src/SimpleViewer.cpp

```diff
@@ -783,6 +797,177 @@ void SimpleViewer::changeAllVisibility(utils::Entity entity, bool changeToVisibl
     }
 };
 
+static const char* formatToName[] = {
+    "R8",
+    "R8_SNORM",
+    "R8UI",
```

Comment:
A switch-case on `filament::Texture::InternalFormat` instead would guard against upstream changes in the enum

Reply:
@gaborvalasek  you mentioned you were concerned about the bad performance of switch/case. Godbolt suggest that Clang 13 can optimize it to a LUT. Not that it matters. I think guarding against upstream changes outweighs any possible performance difference here.

Reply:
I've checked this as well and as long as you have no large gaps in your cases, it indeed LUTs it up. Since these IDs are continuous, I'm OK with making it into an implicit LUT. Fixed in https://github.com/shapr3d/filament/pull/20/commits/ddf94a75580b2a0573fd9d18dc5bc78c44e25463